Source/WebInspectorUI/UserInterface/Models/MemoryTimelineRecord.js

```diff
@@ -83,8 +83,10 @@ WI.MemoryTimelineRecord = class MemoryTimelineRecord extends WI.TimelineRecord
             memoryCategories.push({type: WI.MemoryCategory.Type.JavaScript, size: javascriptSize});
         if (imagesSize)
             memoryCategories.push({type: WI.MemoryCategory.Type.Images, size: imagesSize});
-        if (layersSize)
-            memoryCategories.push({type: WI.MemoryCategory.Type.Layers, size: layersSize});
+
+        // Memory usage of "Layers" can be "0"
+        memoryCategories.push({type: WI.MemoryCategory.Type.Layers, size: layersSize});
```

Comment:
what is the motivation/goal of this change?

also, i think this means that _every_ `WI.MemoryTimelineRecord` will now have a `WI.MemoryCategory.Type.Layers` since `layersSize` defaults to `0` at the top, which is probably not what we want.  IMO if a `WI.MemoryTimelineRecord` has a `WI.MemoryCategory.Type.Layers` with a `size` of `0` then we probably dont want/need to show it since there's nothing to show as it is 0 after all (i.e. having an empty graph is arguably worse than not having a graph as it could draw attention where none is deserved)

Reply:
Sure, I will undo it and create a separate bug report about that. I completely agree with you that this is not related to this PR, but this change will depend on the one about described issue.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=259553